app/src/pages/inside/testPlansPage/testPlanDetailsPage/testPlanDropZones/testPlanDropZones.scss

```
@@ -0,0 +1,109 @@
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+.test-plan-drop-zones {
+  --side-panel-width: 560px;
+  --app-sidebar-width: 48px;
+  --overlay-z-index: var(--rp-ui-base-z-index-overlay-panel);
+  position: fixed;
```

Stylelint (17.9.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)